site/_includes/scripts.html

```diff
@@ -80,7 +83,50 @@
 
   $('#collapsing-navbarHead').megamenu();
 
+  // Swiper pause and play
+  $(document).ready(function () {
+  var MySwiper = document.querySelector('.swiper-container').swiper;
```

Comment:
and what's append on pages where there's no swiper container?
=> a JS error
- check before if `.swiper-container` element exist onto the page please

Reply:
oh waoh! I forget that... It was suposed to be inside the click event function.